devicetypes/smartthings/zigbee-ceiling-fan-light.src/led-fan-lightings.groovy

```diff
@@ -76,7 +76,7 @@ def parse(String description) {
 				it.device.deviceNetworkId == "${device.deviceNetworkId}:1" 
 			}          
 			event.displayed = true
-			zigbeeMap.name = "fanSpeed"
+			zigbeeMap.name = "fanSpeedEvent"
```

Comment:
I've meant more like
```
def fanSpeedEvent = createEvent(name: "fanSpeed", value: zigbeeMap.value)
child.sendEvent(fanSpeedEvent)
```
The way you changed it, it won't work at all. And remember also about looking at this comment - https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/68445#discussion_r687482840.

Reply:
Thank you very much :) I fixed it.